gaek/ndb_json.py

```diff
@@ -67,14 +67,21 @@ def encode_generator(obj):
   return list(obj)
 
 
-def encode_key(obj):
+def encode_key_as_entity(obj):
   """Get the Entity from the ndb.Key for further encoding."""
   # NOTE(erichiggins): Potentially poor performance for Models w/ many KeyProperty properties.
-  return obj.get_async()
-  # Alternative 1: Convert into pairs.
-  # return obj.pairs()
-  # Alternative 2: Convert into URL-safe base64-encoded string.
-  # return obj.urlsafe()
+  # NOTE(ronufryk): Potentially can cause circular references and "RuntimeError: maximum recursion depth exceeded"
+  return obj.get()
```

Comment:
Could this stay as `.get_async` to help with performance issues? I believe the the `ndb.Future` type should resolve it to an actual entity.